app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +240,18 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      {if @state.workflows.length > 0
+        meta = @state.workflows[0].getMeta()
+
+        <Paginator
+          className="lab-visibility-paginator"
```

Comment:
Yeah, this a bit of my bad after taking a closer look again. There are a few styles that probably should move into the paginator stylus file as base styles for the buttons.